tests/common.bash

```diff
@@ -137,6 +137,10 @@ function apply_cluster_admission_policy {
 	wait_for_cluster_admission_policy PolicyUniquelyReachable
 }
 
+function apply_cluster_admission_policy_with_no_wait {
```

Comment:
I would prefer to remove this function in favor of using plain `kubectl apply -f` in code.